python/paddle/fluid/__init__.py

```diff
@@ -175,6 +175,7 @@ def __bootstrap__():
         read_env_flags.append('communicator_max_merge_var_num')
         read_env_flags.append('communicator_fake_rpc')
         read_env_flags.append('communicator_send_wait_times')
+        read_env_flags.append('communicator_merge_sparse_grad')
```

Comment:
maybe you should add flag document for this flag. I'm ok with this flag, do you have any recommendation about how to set this value?